lnrpc/walletrpc/walletkit_server.go

```diff
@@ -1273,6 +1273,13 @@ func (w *WalletKit) SignPsbt(_ context.Context, req *SignPsbtRequest) (
 		return nil, fmt.Errorf("error serializing PSBT: %v", err)
 	}
 
+	// Check if we actually signed any inputs by comparing FundedPsbt
+	// and SignedPsbt. If there is no difference we signal this to
+	// the user.
+	if bytes.Equal(req.FundedPsbt, signedPsbtBytes.Bytes()) {
```

Comment:
I don't think this should be an error... Since it could be useful to just check if any key is being controlled by `lnd` by calling this RPC. How about we add a new field `inputs_signed` that is a list of all indices that `lnd` added a signature for?

Reply:
Thank you for the review! I agree about this being an error. I was going with the suggestion in the attached issue. Listing the signed inputs is neat. The caller can decide what to do if the list returned is empty. Will work on it in a bit.

Reply:
@guggero just to confirm the approach. Are you suggesting that we add the index array to `psbt.Packet` and record the index of the input that was signed here: https://github.com/lightningnetwork/lnd/blob/fec8fd9c63dc672dc3ea1b9be47bd1d6c4a0a54d/lnwallet/btcwallet/psbt.go#L179 ?

Reply:
@guggero are you suggesting this new index array to live in the `SignPsbtResponse` or in the `psbt.Packet`?

Reply:
`SignPsbtResponse`. Anything in `psbt.Packet` is defined in BIP-174 so we can't just add stuff to it.